cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/HBase96TableUtil.java

```diff
@@ -255,6 +256,28 @@ public BloomType getBloomFilter(HColumnDescriptor columnDescriptor) {
     return datasetStat;
   }
 
+  @Override
+  public String getSysConfigTablePrefix(String tableName) {
+    TableName hTableName = TableName.valueOf(tableName);
+    String rootPrefix;
+    if (Constants.DEFAULT_NAMESPACE.equals(hTableName.getNamespaceAsString())) {
+      // Input: cdap.table (in default namespace)
+      // Output: cdap_system:
```

Comment:
I had meant that 'cdap_system' will be the output of the method given that input.
The `_system:` component is added at the bottom of the method.